scripts/template_upload_rhevm.py

```diff
@@ -630,6 +675,10 @@ def upload_template(rhevip, sshname, sshpass, username, password,
             logger.info("RHEVM:%r Found finished template with name %r.", provider, template_name)
             logger.info("RHEVM:%r The script will now end.", provider)
             return True
+        logger.info('RHEVM:%r BEFORE add_glance..', provider)
```

Comment:
**Required**

Discussed offline - going to remove/comment out the call to add_glance and sys.exit that was for debugging.

Going to merge this functionality and open other PR's for using this/removing OVA upload.